src/calendar-list/index.js

```diff
@@ -283,6 +283,9 @@ class CalendarList extends Component {
           data={this.state.rows}
           //snapToAlignment='start'
           //snapToInterval={this.calendarHeight}
+          viewabilityConfig={{
+            itemVisiblePercentThreshold: 50
+          }}
```

Comment:
Currently the viewabilityConfig sets itemVisiblePercentThreshold to 20. I don't see how that or changing it to 50 will fix the issue with invoking onMonthChange

Reply:
https://stackoverflow.com/questions/45868284/how-to-get-currently-visible-index-in-rn-flat-list

Reply:
Its wark i checked

Reply:
onVisibleMonthsChange={(months) => { console.log('now these months are visible', this.setMonths(months)); }}
          pagingEnabled={true}
 result =>{year: 2019, month: 2, day: 19, timestamp: 1550534400000, dateString: "2019-02-19"}

Reply:
u can checked too

Reply:
we need current month , but now  if(pagingEnabled={true})  
result => #659

Reply:
First, setting itemVisiblePercentThreshold to 50 is will give you the same behaviour as the FlatList's default. Second, we already have 'this.viewabilityConfig' defined in the c'tor and you're ignoring it. And last creating a new object in the render function is bad practice. Please run the library again and check if your fix is still relevant

Reply:
U are right in last version all wark thanks!!!  i warked width version 1 . 68

Reply:
That's great. Can you please close the PR